test/ArtifactApplication.js

```diff
@@ -85,123 +85,123 @@ contract('ArtifactApplication', async accounts => {
+    //       accounts[0],
+    //       ARTIFACT.artist,
+    //       ARTIFACT.metaUri,
+    //     );
+    //     await governance.approve(0);
+    //     await registry.transfer(from, to, tokenId, metaUri, price, location, date, false);
+
+    //     const result = await governance.getARRLength();
+
+    //     expect(result.toNumber()).to.be.eql(0);
+    //   });
+
+    //   describe('ARR calculation', async () => {
+    //     const assertARRCalculationCorrectForPrice = function (price, expectedARR) {
+    //       it('calculates ARR correctly for €' + price / 100, async () => {
+    //         await artifactApplication.applyFor(
+    //           accounts[0],
+    //           ARTIFACT.artist,
+    //           ARTIFACT.metaUri,
+    //         );
+    //         await governance.approve(0);
+    //         await registry.transfer(from, to, tokenId, metaUri, toBN(price), location, date, true);
+    //         const result = await artifactApplication.getARR(0);
+    //         const actualARR = result[4];
+    //         expect(actualARR.toNumber()).to.be.eql(expectedARR);
+    //       });
+    //     };
+
+    //     context('when sale price between €0 and €999.99', async () => {
+    //       assertARRCalculationCorrectForPrice(99999, 0);
+    //     });
+
+    //     context('when sale price between €1000 and €50000', async () => {
+    //       assertARRCalculationCorrectForPrice(100000, 4000);
+    //       assertARRCalculationCorrectForPrice(3042124, 121685);
+    //       assertARRCalculationCorrectForPrice(5000000, 200000);
+    //     });
+
+    //     context('when sale price between €50000.01 and €200000', async () => {
+    //       assertARRCalculationCorrectForPrice(5000001, 200000);
+    //       assertARRCalculationCorrectForPrice(6505043, 245151);
+    //       assertARRCalculationCorrectForPrice(6505063, 245152);
+    //       assertARRCalculationCorrectForPrice(20000000, 650000);
+    //     });
+
+    //     context('when sale price between €200000.01 and €350000', async () => {
+    //       assertARRCalculationCorrectForPrice(20000001, 650000);
+    //       assertARRCalculationCorrectForPrice(20178652, 651787);
+    //       assertARRCalculationCorrectForPrice(35000000, 800000);
+    //     });
+
+    //     context('when sale price between €350000.01 and €500000', async () => {
+    //       assertARRCalculationCorrectForPrice(35000001, 800000);
+    //       assertARRCalculationCorrectForPrice(49685500, 873428);
+    //       assertARRCalculationCorrectForPrice(50000000, 875000);
+    //     });
+
+    //     context('when sale price is above €500000', async () => {
+    //       assertARRCalculationCorrectForPrice(50000001, 875000);
+    //       assertARRCalculationCorrectForPrice(80085423, 950214);
+    //       assertARRCalculationCorrectForPrice(200000000, 1250000);
+    //       assertARRCalculationCorrectForPrice(111110085423, 1250000);
+    //     });
+    //   });
+    // });
```

Comment:
Surely should be copy pasted to a new test file for the contract that now does this functionality not just commented out?!